src/modules/main/components/MainExpertsView.tsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { ExpertBlock } from '../../../lib/components/ExpertBlock';
 import { useStyles } from '../styles/MainExpertsView.styles';
-import cards from '../mockDataExperts';
+import mockCards from '../mockDataExperts';
+import { loadExperts } from '../../direction/store/actions';
+import { IRootState } from '../../../store/rootReducer';
 
 const cardsClasses = Array.from(Array(11).keys()).map((el) => `item_${el}`);
+const selectExperts = (state: IRootState) => state.main.experts;
 
 export const MainExpertsView: React.FC = () => {
+  const dispatch = useDispatch();
+  dispatch(loadExperts(mockCards));
```

Comment:
same as https://github.com/ita-social-projects/dokazovi-fe/pull/38
- pls load data in the redux action

Reply:
don't pass the data as a func arguments